odata-mock/src/server/types.ts

```diff
@@ -0,0 +1,13 @@
+
+type apiType = {
+    name: string,
+    metadata: string,
+    specification: string,
+    type: string
+}
+type VarkesConfigType = {
+    name: string,
+    apis: apiType[]

```

Comment:
add events also to VarkesConfigType
Also if one of the elements don't exist in the file will that cause a problem?

Reply:
added